rules/windows/powershell/powershell_alternate_powershell_hosts.yml

```diff
@@ -19,10 +19,7 @@ falsepositives:
 level: medium   
 detection:
     filter:
-        - ContextInfo: 'powershell.exe'
-        - Message: 'powershell.exe'
-        # Both fields contain key=value pairs where the key HostApplication is relevant but
-        # can't be referred directly as event field.
+        ContextInfo: 'powershell.exe'
     condition: selection and not filter
```

Comment:
Not quite. As written in the comment, the ContextInfo field contains a number of key=value pairs, e.g.:

`Severity = Informational Host Name = ConsoleHost Host Version = 5.1.14393.3383 Host ID = d72f218b-82c7-499c-97cb-c80e944516ab Host Application = C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe Engine Version = 5.1.14393.3383 Runspace ID = 4ecdbb14-73d6-41a0-8b4a-662d7fa34856 Pipeline ID = 84 Command Name = Set-StrictMode Command Type = Cmdlet Script Name = C:\Program Files\WindowsPowerShell\Modules\PSReadline\1.2\PSReadLine.psm1 Command Path = Sequence Number = 884 User = OFFSEC\admmig Connected User = Shell ID = Microsoft.PowerShell`

However, unless the backend parses this entry manually, this will simply be the text in the ContextInfo field and we can't access the "embedded" HostApplication field with Sigma.

Reply:
I agree , It is why original rule had the `Message: 'powershell.exe'` in it.
So the rule EventID=400 AND ContextInfo='*' will never match

Reply:
You mean, if the backend splits the ContextInfo field into its key=value pairs?

Reply:
It is not the backend but the log parser (winlogbeat, logstash ,..) many rule use `ContextInfo: value` or `keyword :  -'ContextInfo=value' ` need a good revue of all rules 😄

Reply:
The fast way , is to remove this rule from the PR so I can merge your works.
You or I works on powershell_alternate_powershell_hosts.yml in another PR to fix

Reply:
One interesting point is that other rules (e.g. rules/windows/powershell/powershell_suspicious_download.yml) refer to this as HostApplication, not "Host Application". Maybe something with the different Event IDs, 400 vs. 4103, or specific log parser behaviour? Something that needs to be investigated for the future PR.